prov/ucx/src/ucx_domain.c

```diff
@@ -229,6 +264,9 @@ static int ucx_mr_regattr(struct fid *fid, const struct fi_mr_attr *attr,
 	um_params.length = attr->mr_iov->iov_len;
 	um_params.flags = 0;
 
+	ucx_update_memtype_cache(attr->iface, attr->mr_iov->iov_base,
```

Comment:
attr->iface may not be initialized.  See ucx_mr_regv

Reply:
Thanks. Updated.